Update artifact tarball when `spec.ignore` changes (#712)
fluxcd/source-controller: docs/spec/v1beta2/buckets.md

the same time, for example due to a newly introduced configuration issue in the
Bucket spec.

### Ignore Checksum

> The source-controller calculates the SHA256 checksum of the last Bucket's
> [`.spec.ignore` field](#ignore) and record it in the `.status.ignoreChecksum` field. This field
> is used to detect `.spec.ignore` updates to allow artifact changes.

Reviewer: ```suggestion
The source-controller calculates the SHA256 checksum Bucket's [`.spec.ignore` field](#ignore),
and records it in `.status.ignoreChecksum`. This field is used to detect `.spec.ignore` changes to
trigger an Artifact update.
```
